components/fxa-client/src/account.rs

```diff
@@ -19,8 +19,6 @@ impl FirefoxAccount {
     /// Get the token server URL
     ///
     /// The token server URL can be used to get the URL and access token for the user's sync data.
-    ///
-    /// **💾 This method alters the persisted account state.**
```

Comment:
They still will, but the plan is to have them call the `FxaStorage.save_state()` method which will handling saving the state.

That said, it's not like the docstring is wrong so we could keep it around.  I removed them because I thought the only reason for these messages was to inform the consumer that they needed to call `to_json()` and save the results after the method call.

Reply:
Maybe these comments should be updated to say "will callback into the app-supplied storage handler to persist a new state"?

(TBH I'm not that bothered by the comments existing or not, just trying to get my head around the model)